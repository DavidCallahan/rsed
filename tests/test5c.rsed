print "#!/bin/bash -x"
constant = "foo"
foreach all
    if "(.*)/TARGETS-" then
       $dir = $1
       if "name\s*=\s*'(.*)'" then
           print "./runbench.sh " "in" $dir ":" $1 " _bin/" $constant $dir "/" $1 "a" "b"
       end 
    end
    skip
end
