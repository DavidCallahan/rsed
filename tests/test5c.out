#!/bin/bash -x
./runbench.sh inadmarket/adsearch2/query:queryengine_benchmark _bin/fooadmarket/adsearch2/query/queryengine_benchmarkab
./runbench.sh inadmarket/adsearch2/query:queryengine_benchmark_custom _bin/fooadmarket/adsearch2/query/queryengine_benchmark_customab
./runbench.sh inadmarket/asx/common/test:ngram_bench _bin/fooadmarket/asx/common/test/ngram_benchab
./runbench.sh inadmarket/cpe/common/tests:cpe_laser_client_benchmark_test _bin/fooadmarket/cpe/common/tests/cpe_laser_client_benchmark_testab
